ext/datasource/helper.go

```diff
@@ -92,16 +75,8 @@ func SystemRulesJsonConverter(src []byte) (interface{}, error) {
 	}
```

Comment:
In order to keep naming consistent.
Please rename function name, such as: SystemRuleJsonArrayParser.